go.mod

```
@@ -0,0 +1,10 @@
+module PhilomathesInc/demo-ci-pipelines
```

```suggestion
module github.com/PhilomathesInc/demo-ci-pipelines
```
Import paths without dots in the first path element are reserved for the standard library and toolchain.

See - https://github.com/golang/go/issues/32819, also - https://golang.org/ref/mod

---

Although you might get away with using this for now, it is not standard, and might break with any future release of go, since this behaviour is not promised.

---

Yes, will fix it. Thanks for pointing out.